### DISCUSSION TITLE
I'm having trouble getting the app to send an SMS to my Google phone number

### DISCUSSION THREAD

**Discussion by MotorCityCobra** [jef/streetmerchant#1792] [Help]
In my `dotenv` file I have...   

`PHONE_CARRIER='google'`
`PHONE_NUMBER='19145559620'` (fake number for public but real number in code)
I tried writing the phone number string with double quotes too.

I have successfully configured email, which is also through Google by turning off my Google account's two factor authentication but SMS is not sending.

Before turning off two factor authentication I got error logs for both email and SMS but now it says  

`[2:52:01 PM] info :: ✔ email sent`
`[2:52:01 PM] info :: ✔ sms sent`

Though as I said, I get the email but I do NOT get any SMS to my number.

This Google Phone number is my only option.

**Comment by NikkixSage** [comment]
EMAIL_PASSWORD=(This is where you have your app password)
EMAIL_TO=(App password that is connected to your gmail account)
EMAIL_USER=(Same as EMAIL_TO=XxYZxX@gmail.com <--- example)
PHONE_CARRIER=your phone carrier you're trying to push SMS
PHONE_NUMBER= Self explanatory (Try with area code and without area code, mine worked with my area code)

**Comment by jef** [comment]
Is it Google Fi or Google Voice?
